.github/workflows/checkimage.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
           base=$(grep -Po '(?<=FROM )([^\s]*)(?= AS build)' Dockerfile)
           skopeo inspect "docker://$base" | jq .Digest --raw-output > .baseimagedigest
           docker run --rm -u 0 quay.io/cloudservices/compliance-backend:latest sh -c \
-            'microdnf update -y $(cat /opt/installedpackages) > /dev/null; rpm -q $(cat /opt/installedpackages) | sort | sha256sum | cut -d " " -f 1' \
+            'microdnf update -y $(rpm -qa | grep -v 'gpg-pubkey') > /dev/null; rpm -q $(rpm -qa | grep -v 'gpg-pubkey') | sort | sha256sum | cut -d " " -f 1' \
```

Comment:
Wouldn't it be better to update the `/opt/installedpackages` file or use a parameter that would hold list of packages that are being installed outside of the base for this automation to track only those specific ones, not all?